jupyterlab_hdf/util.py

```diff
@@ -329,7 +331,7 @@ def jsonize(v):
         return [v.real, v.imag]
     if isinstance(v, h5py.Empty):
         return None
-    return v
+    raise TypeError("Cannot jsonize {}".format(type(v)))
```

Comment:
I had to change `jsonize` a bit to fit the expected behaviour of default: https://github.com/ijl/orjson#default